2023-05-15 14:49:32,818 - ERROR - {'while connecting to MongoDB', 'Error:'}
2023-05-15 14:49:33,010 - ERROR - {'Error:', 'while connecting to MongoDB'}
2023-05-15 17:50:29,284 - ERROR - {'Error:', 'while connecting to MongoDB'}
2023-05-15 18:55:07,234 - ERROR - {'Error': "'Email_handler' object has no attribute 'find_total'"}
2023-05-15 22:20:51,805 - ERROR - {'while connecting to MongoDB', 'Error:'}
2023-05-15 22:21:13,409 - ERROR - {'while connecting to MongoDB', 'Error:'}
2023-05-15 22:21:28,237 - ERROR - {'Error:', 'while connecting to MongoDB'}
2023-05-15 22:22:05,166 - ERROR - {'Error:', 'while connecting to MongoDB'}
2023-05-15 22:22:15,814 - ERROR - {'Error:', 'while connecting to MongoDB'}
2023-05-15 22:23:02,397 - ERROR - {'Error:', 'while connecting to MongoDB'}
2023-05-15 22:23:49,365 - ERROR - {'while connecting to MongoDB', 'Error:'}
2023-05-15 22:24:39,834 - ERROR - {'Error:', 'while connecting to MongoDB'}
2023-05-15 22:55:18 - ERROR  - [MainThread:<module>():13] - {'while connecting to MongoDB', 'Error:'}
2023-05-15 23:01:10,950 - ERROR - {'while connecting to MongoDB', 'Error:'}
2023-05-15 23:03:41,753 - ERROR - mongo.py:13 - <module> - {'Error:', 'while connecting to MongoDB'}
2023-05-15 23:08:41,843 - ERROR - mongo.py:13 - <module> - {'Error:', 'while connecting to MongoDB'}
